### DISCUSSION TITLE
WebViewAuthenticator default client

### DISCUSSION THREAD

**Discussion by theomarie** [sbertix/Swiftagram#138] [Q&A] [answered]
Hello everyone,
I just followed your documentation to authenticate myself from WKwebview, I encounter a strange error.

Authentication is not successful and I receive an email telling me that the connection comes from an Android phone.
Samsung SM-G981B ·

I think the problem is due to the authentication with the facebook profile?

**Comment by sbertix** [comment, marked as answer]
Hey @theomarie, 

This is not an error 😊 
The default `Client` is indeed an Android device. To use an iPhone you should call this immediately after launch…

```swift
Client.default = .current ?? .iPhone11ProMax
```

(You might need wrapping this in `#IF` statements depending on your deployment target)

Future versions will switch to iOS devices by default, but for now, as the feature is still being tested, we keep relying on Android ones.
